shell/package.json

```diff
@@ -12,7 +12,8 @@
         "access": "public"
     },
     "dependencies": {
-        "@dhis2/app-adapter": "7.0.1",
+        "@dhis2/app-adapter": "7.1.0",
+        "@dhis2/sw": "file:../sw",
```

Comment:
This can be updated to use workspace resolution, right?

Reply:
Yes; it needed to be this way for the dashboard app to use this branch locally I think.

Related: when I try to use `yarn add @dhis2/sw@7.1.0` to add the dependency, I get an error:
```sh
```

Reply:
Is that the same way you added the app-service-offline dependency to the runtime?
https://github.com/dhis2/app-runtime/pull/901/commits/ec1590ce8b1dc6f731aecae166faffa9ac904a12